### PR TITLE
Add Spirits Unchained to the Addon Page

### PR DESCRIPTION
**Pull request by JustAHuman-xD**
With the addition of Spirits to the Builder, here is the PR for the wiki page

### CODE REVIEW

File: pages/Addons.md [hunk 1]
```diff
@@ -80,6 +80,7 @@ Wildernether | SchnTgaiSpock | <details><summary>expand</summary>Wildernether is
...
 SlimeHUD | SchnTgaiSpock | <details><summary>expand</summary>SlimeHUD adds a WAILA display similar to WhatIsThis, but with extended funcitonality and customization.</details> | [Readme](https://github.com/SchnTgaiSpock/SlimeHUD/blob/master/README.md) | [Github](https://github.com/SchnTgaiSpock/SlimeHUD) | [Builds](https://thebusybiscuit.github.io/builds/SchnTgaiSpock/SlimeHUD/master/)
+SpiritsUnchained | JustAHuman_xD | <details><summary>expand</summary>Spirits Unchained is an addon focused on a new mechanic, Spirits! Spirits can be found roaming the wild or have a chance at getting release when killing a mob! Players can catch or bottle spirits and discover each types unique effects on its user. But be careful, spirits will not always be friendly. Spirits also has a built in guid book that you can access through the Slimefun guide if you need help using the addon!</details> | [Readme](https://github.com/JustAHuman-xD/SpiritsUnchained/blob/master/README.md) | [Github](https://github.com/JustAHuman-xD/SpiritsUnchained) | [Builds](https://thebusybiscuit.github.io/builds/JustAHuman-xD/SpiritsUnchained/master/)
```
Comment (Sefiraat) [inline pages/Addons.md:83]:
```suggestion
SpiritsUnchained | JustAHuman_xD | <details><summary>expand</summary>Spirits Unchained is an addon focused on a new mechanic, Spirits! Spirits can be found roaming the wild or have a chance at getting released when killing a mob! Players can catch or bottle spirits and discover each types unique effects on its user. But be careful, spirits will not always be friendly. Spirits also has a built in guide book that you can access through the Slimefun guide if you need help using the addon!</details> | [Readme](https://github.com/JustAHuman-xD/SpiritsUnchained/blob/master/README.md) | [Github](https://github.com/JustAHuman-xD/SpiritsUnchained) | [Builds](https://thebusybiscuit.github.io/builds/JustAHuman-xD/SpiritsUnchained/master/)
```

**Review by svr333 — APPROVED** [review]

**Review by Sefiraat — APPROVED** [review]

File: pages/Addons.md [hunk 2]
```diff
@@ -80,6 +80,7 @@ Wildernether | SchnTgaiSpock | <details><summary>expand</summary>Wildernether is
...
 SlimeHUD | SchnTgaiSpock | <details><summary>expand</summary>SlimeHUD adds a WAILA display similar to WhatIsThis, but with extended funcitonality and customization.</details> | [Readme](https://github.com/SchnTgaiSpock/SlimeHUD/blob/master/README.md) | [Github](https://github.com/SchnTgaiSpock/SlimeHUD) | [Builds](https://thebusybiscuit.github.io/builds/SchnTgaiSpock/SlimeHUD/master/)
+SpiritsUnchained | JustAHuman_xD | <details><summary>expand</summary>Spirits Unchained is an addon focused on a new mechanic, Spirits! Spirits can be found roaming the wild or have a chance at getting released when killing a mob! Players can catch or bottle spirits and discover each types unique effects on its user. But be careful, spirits will not always be friendly. Spirits also has a built in guide book that you can access through the Slimefun guide if you need help using the addon!</details> | [Readme](https://github.com/JustAHuman-xD/SpiritsUnchained/blob/master/README.md) | [Github](https://github.com/JustAHuman-xD/SpiritsUnchained) | [Builds](https://thebusybiscuit.github.io/builds/JustAHuman-xD/SpiritsUnchained/master/)
```
Comment (Phoenix-Starlight) [inline pages/Addons.md:83]: Possessive missing? Perhaps types' (type's?) or spirit's

**Review by Phoenix-Starlight — CHANGES_REQUESTED** [review]

File: pages/Addons.md (hunk 2 above)
Comment (JustAHuman-xD) [reply pages/Addons.md:83]: Will do uno momento

**Review by svr333 — DISMISSED** [review]

**Review by svr333 — APPROVED** [review]

**Review by Phoenix-Starlight — APPROVED** [review]
*Stamps it*

**Review by svr333 — APPROVED** [review]

**Pull request merged**